magicclass/ext/vtk/__init__.py

```diff
@@ -8,6 +7,6 @@
 
 """
 
-from .widgets import VtkCanvas
+from .widgets import VedoCanvas
 
-__all__ = ["VtkCanvas"]
+__all__ = ["VedoCanvas"]
```

Comment:
I've also been thinking that the name should be "Vedo*" instead of "Vtk*"... could you just allow the alias "VtkCanvas" and add a deprecation warning on importing `VtkCanvas` using `__getattr__` just like magicclass.__init__.py, please?

Reply:
I tried to follow the example of other deprecations in `__getattr__` but i'm not sure why that did not seem to work so i've done it the dumb way of subclassing the VedoCanvas.. maybe you can have a look! :)